src/h5web/visualizations/shared/TooltipMesh.tsx

```diff
@@ -65,11 +65,22 @@ function TooltipMesh(props: Props): ReactElement {
   const onPointerOut = useCallback(hideTooltip, [hideTooltip]);
   const onPointerDown = useCallback(hideTooltip, [hideTooltip]);
 
+  // Show tooltip after dragging unless pointer has left canvas
+  const onPointerUp = useCallback(
+    (evt: PointerEvent) => {
+      const { offsetX: x, offsetY: y } = evt.nativeEvent;
+      if (x >= 0 && x <= width && y >= 0 && y <= height) {
+        onPointerMove(evt);
+      }
+    },
+    [height, onPointerMove, width]
+  );
+
```

Comment:
Just a quick side fix as explained in the comment. At the moment, you have to move the mouse again after dragging to see the tooltip.